Online-Messaging-App/src/app/register/register.component.html

```diff
@@ -1,29 +1,57 @@
 <div class="container">
-
+  <mat-toolbar color="primary" class ="navbar">
+    <div>Streamline Athletes</div>
+    <div>
+      <span>Home</span>
+      <span>About</span>
+      <span>Services</span>
+    </div>
+  </mat-toolbar>
+  <div align="center" class="mat-display-2">Streamline Athletes</div>
   <div class="row">
     <div class="col-lg-4">
       <form (ngSubmit)="registerSubmit(regform)" #regform="ngForm">
-        <div class="form-group">
-          <label for="username">Username</label>
-          <input type="text" name="username" id="username" ngModel class="form-control">
+        <div align = "center" class="form-group">
+          <mat-form-field appearance="fill" color = "primary">
+            <mat-label>Username</mat-label>
+            <input matInput type="text" name="username" id="username" ngModel class="form-control">
+          </mat-form-field>
+          <mat-form-field appearance="fill" color = "primary">
+            <mat-label>First Name</mat-label>
+            <input matInput type="text" name="firstName" id="firstName" ngModel class="form-control">
+          </mat-form-field>
         </div>
-        <div class="form-group">
-          <label for="password">Password</label>
-          <input type="password" name="password" id="password" ngModel class="form-control" required>
-        </div>
-        <div class="form-group">
-          <label for="email">Email</label>
-          <input type="email" name="email" id="email" ngModel class="form-control">
+
+        <div align = "center" class="form-group">
+          <mat-form-field appearance="fill" color = "primary">
+            <mat-label>Password</mat-label>
+            <input matInput type="password" name="password" id="password" ngModel class="form-control">
+            <mat-hint align = "end">Minimum 8 Characters</mat-hint>
```

Comment:
Just keep in mind I this may change/become more complex

Reply:
Done